src/Operations/CreateCustomContour.cc

```diff
@@ -10,6 +10,8 @@
 
 #include "CreateCustomContour.h"
 
+using namespace std;
```

Comment:
not sure if this is important, but Hal's convention is using std:: instead of having this at the top